lib/mirage.ml

```diff
@@ -930,7 +205,7 @@ module Project = struct
   let packages = [package "mirage"]
 
   (* The directories to ignore when compiling config.ml *)
-  let ignore_dirs = ignore_dirs
+  let ignore_dirs = Mirage_configure_solo5.ignore_dirs
```

Comment:
this is dead code afaict (the `ignore_dirs` are used elsewhere but not in the Mirage module anymore)

Reply:
CI claims you still have this line in mirage.ml and it leads to compilation error..
```suggestion

```